css/style.css

```diff
@@ -0,0 +1,43 @@
+* {
+}
+body {
+}
+
+#title,
+#description {
+  text-align: center;
+}
+#title {
+  margin-top: 10vh;
+}
```

Comment:
it is best practice to have a new line after each class selector. Please add new line below this class selector